### PR TITLE
Optionally truncate long metric names

### PR DESCRIPTION
**Pull request by AeroNotix**
* Some (HostedGraphite) will close connections with metric names over
  1kb (don't ask ;))

### CODE REVIEW

File: src/folsomite_server.erl [hunk 1]
```diff
@@ -182,6 +188,10 @@ get_system_info() ->
     [{K, V} || {K, V} <- L,
                lists:member(K, [port_count, process_count])].
 
+truncate_metric_name(Metric, Length) ->
+    Format = lists:flatten(io_lib:format("~~-~Bs~n", [Length])),
```
Comment (AeroNotix) [inline src/folsomite_server.erl]: Better way to do the runtime sized string truncation with io_lib:format?

Comment (puzza007) [reply src/folsomite_server.erl]: Seems reasonable to me

**Comment by puzza007** [comment]
Travis failure seems spurious

```
===> Failed to write package index.
```

File: src/folsomite_server.erl (hunk 1 above)
Comment (puzza007) [reply src/folsomite_server.erl]: Should this be `Length - 1` because of the `\n`?

Comment (AeroNotix) [reply src/folsomite_server.erl]: Great point! Can change

**Pull request merged**